test/factories/discovery_rule_related.rb

```diff
@@ -10,4 +10,15 @@
     organizations { [organization_one] } if SETTINGS[:organizations_enabled]
     locations { [location_one] } if SETTINGS[:locations_enabled]
   end
+
+  factory :another_discovery_rule, class: DiscoveryRule do
+    sequence(:name) { |n| "rule#{n}" }
+    sequence(:priority) { |n| n }
+    search { "facts.fact1 == 'value1'" }
+    hostname { "" }
+    max_count { 0 }
+    enabled { true }
+    organizations { [organization_two] } if SETTINGS[:organizations_enabled]
+    locations { [location_two] } if SETTINGS[:locations_enabled]
+  end
```

Comment:
Please use some name like `discover_rule_in_organization_two`. But `trait` could be used for that I guess.